cmd/kube-apiserver/app/server.go

```diff
@@ -243,7 +245,15 @@ func Run(s *options.APIServer) error {
 	if err != nil {
 		glog.Errorf("Failed to create clientset: %v", err)
 	}
-	admissionController := admission.NewFromPlugins(client, admissionControlPluginNames, s.AdmissionControlConfigFile)
+
+	namespaceInformer := informers.CreateSharedNamespaceIndexInformer(client, 5*time.Minute)
```

Comment:
I think this ResyncPeriod should be fuzzed similar to controller MinResyncPeriod